package/yast2-installation-control.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon May 15 10:49:47 UTC 2017 - igonzalezsosa@suse.com
+
+- Make the partitioning elements section reusable
```

Comment:
Yes, you're right. So finally I've decided to branch this package (against `storage-ng` branch) because this change it's not needed by now in `master`.